runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkJobServerDriver.java

```diff
@@ -58,6 +58,12 @@
         usage = "The location to store staged artifact files"
     )
     private String artifactStagingPath = "/tmp/beam-artifact-staging";
+
+    @Option(
+        name = "--flink-job-master-url",
```

Comment:
Should this be called --flink-master-url?

Reply:
Makes sense.